api/src/main/java/com/epam/pipeline/dao/pipeline/PipelineRunDao.java

```diff
@@ -449,11 +450,9 @@ public void deleteRunSidsByPipelineId(final Long pipelineId) {
                 .findFirst();
     }
 
-    public List<PipelineRun> loadServerlessRunsToStop(final LocalDateTime maxLastUpdate) {
-        final MapSqlParameterSource params = new MapSqlParameterSource();
-        params.addValue("MAX_LAST_UPDATE", maxLastUpdate);
+    public List<StopServerlessRun> loadServerlessRunsToStop() {
         return ListUtils.emptyIfNull(getNamedParameterJdbcTemplate()
```

Comment:
I'd suggest to move this method to `StopServerlessRunDao` class

Reply:
Thanks, done